plotsearch/utils.py

```diff
@@ -638,7 +620,7 @@ def initialize_area_search_form():
     Field.objects.create(
         section=invoice_section2,
         label="Postitoimipaika",
```

Comment:
Could you fix this typo now that you are making changes?

Reply:
Absolutely, fixed (it moved to another file)